.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,79 @@
+name: Release from tags
+# Create a GitHub release when a tag is pushed.
+
+on:
+  push:
+    tags:
+      # this is a glob, not a regexp
+      - 'anaconda-*'
+
+permissions:
+  contents: write
+
+jobs:
+  release-from-tag:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Extract version from the tag
+        id: get_version
+        # If the version does not fit the regexp, we'll just fail. TODO maybe abort instead?
+        run: |
+          echo "Checking pushed tag ref ${{ github.ref }}"
+          VER=$(echo ${{ github.ref }} | perl -ne '/^refs\/tags\/anaconda-([0-9]+(?:\.[0-9]+){1,3})-1$/ && print "$1\n";')
```

Comment:
Honestly, I would like to have `awk` or `sed` than `perl`, just because it's yet another way to do the same.

Reply:
This is the variant that maximizes (correct AND compact AND readable).

However if you need it:

```
if echo ${{ github.ref }} | grep -E '^refs/tags/anaconda-([0-9]+(\.[0-9]+){1,3})-1$' ; then
  VER=$(echo ${{ github.ref }} | awk -F - '{print $2}')
  echo "Tag ref ${{ github.ref }} detected as release version $VER."
  echo "::set-output name=version::$VER"
else
  echo "Tag ref ${{ github.ref }} is not a valid release tag."
  exit 1
fi
```